webapp/src/client/app/modules/web-client/ard/web-client-ard.component.ts

```diff
@@ -340,6 +341,8 @@ export class WebClientArdComponent extends WebClientBaseComponent implements OnI
       configBuilder.withExtension(ardQualityMode(connectionParameters.ardQualityMode));
     }
 
+    configBuilder.withExtension(wheelSpeedFactor(connectionParameters.wheelSpeedFactor));
```

Comment:
nitpick: You can probably move that above with all the other unconditional `with`

Reply:
Fixed